examples/storage/couchdb/README.md

```diff
@@ -0,0 +1,84 @@
+
+## Couchdb 2.0.0 on Kubernetes
+This document describes the steps to create a Couchdb 2.0.0 cluster on Kubernetes.
+
+### Prerequisites
+Access to a kubernetes cluster running v1.4 or later, kubectl installed and
+configured.  Minikube was used for this example. Note that the service type
```

Comment:
There is only really one difference for those who are not running minikube and thats how you get the external IP address of the service.  I have updated the README and mentioned how you go about getting the external IP of the service using `kubectl`